config-examples/peer-b-config.toml

```diff
@@ -0,0 +1,18 @@
+public_key = "peer-b-public-key"
+secret_key = "peer-b-secret-key"
+listen = ["[::]:10002"]
+verbosity = "Quiet"
```

Comment:
Is that case sensitive?

Reply:
I think so. I believe we should make this be the different log `levels`, `trace`, `debug`, `info`, `warn`, `error`, `true` -> `info`, `false` -> `warn`,`verbose`-> `debug`

Reply:
Ok sounds good. Can we make it case insensitive?